src/components/AppBar/AppBar.js

```diff
@@ -22,31 +22,30 @@ import {
 } from 'react-share';
 import { STATES } from 'enums';
 
-const _AppBar = () => {
-  const history = useHistory();
-  const { pathname } = useLocation();
-  const panels = ['/', '/faq', '/endorsements', '/contact', '/blog'];
-
-  const getDefaultPanelId = () => {
-    const defaultPanelIndex = Number(panels.indexOf(pathname));
+const Panels = ['/', '/faq', '/endorsements', '/contact', '/blog'];
 
-    if (!defaultPanelIndex) {
-      return 0;
-    }
-
-    // We are on the state page, don't highlight a tab
-    if (defaultPanelIndex === -1) {
-      return false;
-    }
+function getPanelIdxFromLocation(location) {
+  let idx = Panels.indexOf(location.pathname);
+  return idx === -1 ? false : idx;
+}
```

Comment:
tldr: These items were moved up out of `App` because the linter told me to.

Putting both of them inside the function gives the following warning:
![image](https://user-images.githubusercontent.com/1570168/78660452-f41ee080-7881-11ea-8180-c525a1a57447.png)

Adding `getPanelIdxFromLocation` as an effect dependency caused the effect hook to run on every re-render (and also created a warning saying that that would happen).

I realized the moving both `Panels` and `getPanelIdxFromLocation` statically enforces that they won't change between renders.